scripts/out/ctype-schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://kilt-protocol.org/draft-01/ctype#",
+  "$schema": "ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/",
```

Comment:
same

Reply:
how come this does not have an id?

Reply:
bc this is before hashing ?

Reply:
It is a before and after of what a CType should look like. 

Therefore only defining what properties. I think it isn't strictly necessary for the `$schema`to be inside as it will take the title and properties only.

Reply:
Nor including the additional properties.